glossary.md

```diff
@@ -1,7 +1,55 @@
 ## Glossary
 
-* Username
-* Password
-* Session & Session Location
-* Visits
-* Vitals
+**administrative staff**: Individuals who manage people or data in a clinical setting.
+
+**android client**: The OpenMRS mobile client for accessing OpenMRS server instance on Android devices.
+
+**chart**: A graph for patients, measuring height, weight and other observations corresponding to recorded patient vitals.
+
+**clinician**: A doctor, nurse, or other clinical officer who provides health care to patients.
+
+**concept**: The idea that encompasses any question which can be asked about a patient; an observable point of data.
+
+**data**: A piece of knowledge that can be reduced to a single value.
+
+**electronic medical record (EMR)**: A computer system that allows for recording, storage, and retrieval of information related to the delivery of health care to patients.
+
+**encounter**: A clinical transaction between a patient and one or more healthcare providers for the purpose of providing patient services or assessing the health status of the patient. An encounter happens at one point in time. [http://www.astm.org/Standards/E1384.htm](http://www.astm.org/Standards/E1384.htm)
+
+**error**: A message in a computer system that describe a problem currently or recently occurring.
+
+**form**: An electronic form that may be used for entering or viewing data.
+
+**implementer**: Someone who has or is in the process of deploying OpenMRS in a specific location or context of use.
+
+**location**: A physical place where a patient may receive healthcare services.
+
+**metadata**: A piece of information that describes other information.
+
+**module**: A software package that extends OpenMRS functionality in specific ways; often developed by others in the OpenMRS community.
+
+**observation**: One piece of information that is recorded about a person at a moment in time.
+
+**password**: String of characters used for user authentication to prove identity or access approval to gain access to a resource, which is to be kept secret from those not allowed access.
+
+**patient identifier**: Any unique number that can identify a patient. Examples are a Medical Record Number, a National ID, a Social Security Number, a driver's license number, etc.
+
+**patient**: A person receiving health care services.
+
+**person**: Every individual who is referred to in any patient's record in OpenMRS must be stored in the system.
+
+**provider**: A person or company that provides a health care service to you. E.g. doctors, nurses, clinicans or physicians.
+
+**session location**: A provider's current location when inputting data in an EMR encounter. E.g. Outpatient clinic, laboratory.
```

Comment:
Same as above but with session and session location

Reply:
Updating now